winterize/init.lua

```diff
@@ -72,7 +72,7 @@ minetest.register_lbm({
 		node.name ~= "ctf_map:cobble" and minetest.registered_items[node.name].walkable then
 			local found_obstruction = can_snow_fall_freely(pos_above)
```

Comment:
Erm...

Reply:
Oh, you didn't suggest a multi-line change lol

Reply:
Multi-line suggestions haven't yet been implemented ;)

Reply:
I'm 90% sure I've seen the beta feature. The button is kinda hard to find IIRC

Reply:
Multi-line comments are possible, just drag along the line numbers. But the `suggestion` button is greyed out, and hovering over it will display a tooltip that says that the feature isn't available for multi-line comments yet.

Reply:
Figured out what I did. If you made two code suggestions I could have merged both at the same time